src/prototype_creep_startup_tasks.js

```diff
@@ -198,8 +201,8 @@ Creep.prototype.repairStructure = function() {
       for (const spawn of spawns) {
         let found = false;
         let rampart;
-        const structures = spawn.pos.lookFor(LOOK_STRUCTURES);
-        for (const structure of structures) {
+        structures = spawn.pos.lookFor(LOOK_STRUCTURES);
```

Comment:
I think `structures` can stay `const`

Reply:
my ide says no eslint says no

Reply:
better keep it out of for but with a let